src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/PendingBucketDeletionComponent.scala

```diff
@@ -8,14 +8,17 @@ trait PendingBucketDeletionComponent {
   import driver.api._
 
   class PendingBucketDeletionTable(tag: Tag) extends Table[PendingBucketDeletionRecord](tag, "BUCKET_DELETION") {
-    def bucket = column[String]("bucket", O.PrimaryKey)
+    def bucket = column[String]("bucket", O.PrimaryKey, O.Length(254))
 
-    def * = (bucket) <> (PendingBucketDeletionRecord.apply _, PendingBucketDeletionRecord.unapply)
+    def * = bucket <> (PendingBucketDeletionRecord, PendingBucketDeletionRecord.unapply)
   }
 
   object pendingBucketDeletionQuery extends TableQuery(new PendingBucketDeletionTable(_)) {
-    def save(pendingBucketDeletion: PendingBucketDeletionRecord): WriteAction[PendingBucketDeletionRecord] = {
-      pendingBucketDeletionQuery insertOrUpdate pendingBucketDeletion map(_ => pendingBucketDeletion)
```

Comment:
Slick doesn't handle upserts with no non-key columns for MySQL well.  Here, "insert if it doesn't exist" is equivalent behavior.